release-engineering/platforms.md

```diff
@@ -0,0 +1,67 @@
+# Supported Platforms
+
+The Kubernetes release process produces [artifacts](./artifacts.md) for
+different architectures and operating systems. We consider the combination of
+architecture (`GOARCH`) and operating system (`GOOS`) as "platforms". Target of
+this document is to outline different categories of platforms as well as guiding
+through their graduation criteria.
+
+## Tiers
+
+Support for different platforms are organized into three tiers, whereas each
+comes with a different set of guarantees. Tiers can be scoped to single
+binaries or a subset of them. This means for example that we can provide Tier 1
+support for client binaries, even if the server binaries do not exist at all.
+
+### Tier 1
+
+Tier 1 platforms are considered as "guaranteed to work". To achieve this, they
+have to fulfil the following criteria:
+
+- Official binary releases are provided for the platform. This includes
+  container images as well as deb and rpm packages. Building the artifacts is
+  integrated in the release process.
+- Continuous Integration is set up to run tests for the platform. Necessary
+  tests are defined by SIG Release and usually correspond to the
+  [`blocking`](https://testgrid.k8s.io/sig-release-master-blocking) and
+  [`informing`](https://testgrid.k8s.io/sig-release-master-informing) testgrid
+  dashboards.
+- Documentation about the usage of artifacts for the platform is available.
+
+### Tier 2
+
+Tier 2 platforms are considered as "guaranteed to build". To achieve this, they
+have to fulfil the following criteria:
+
+- Official binary releases are provided for the platform. Building the artifacts
+  is integrated in the release process.
+- Automated testing is not or only partially setup.
+
+It may be possible that single features are not available for a certain
+platform.
+
+### Tier 3
+
+Tier 3 platforms are those which the Kubernetes codebase has support for, but
```

Comment:
suggest "codebase has functional support for"

Reply:
Changed as suggested :+1: